roles/network-multus/templates/cni-plugins.yml

```diff
@@ -0,0 +1,43 @@
+---
+apiVersion: extensions/v1beta1
+kind: DaemonSet
+metadata:
+  name: kube-cni-plugins-ds-amd64
+  namespace: {{namespace}}
+  labels:
+    tier: node
+    app: cni-plugins
+spec:
+  template:
+    metadata:
+      labels:
+        tier: node
+        app: cni-plugins
+    spec:
+      hostNetwork: true
+      nodeSelector:
+        beta.kubernetes.io/arch: amd64
+      tolerations:
+      - key: node-role.kubernetes.io/master
+        operator: Exists
+        effect: NoSchedule
+      containers:
+      - name: cni-plugins
+        image: {{cni_provisioner_repo}}:{{cni_provisioner_release}}
+        resources:
+          requests:
+            cpu: "100m"
+            memory: "50Mi"
+          limits:
+            cpu: "100m"
+            memory: "50Mi"
+        securityContext:
+          privileged: true
+        volumeMounts:
+        - name: cnibin
+          mountPath: /opt/cni/bin
+      volumes:
+      - name: cnibin
+        hostPath:
+          path: /opt/cni/bin
+---
```

Comment:
Don't think you need --- at the end. Same comment for all your yaml files.

Reply:
Done